From f966d8648d1a7143b2edc843b1ec99c643475523 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Wed, 10 Feb 2021 15:49:12 +0200
Subject: [PATCH 1486/1921] arm64: dts: fix the Armada 8040 DB AP SDHCI bus
 width

The AP SDHCI on Armada 8040 DB board utilizes 8-bit data lines.

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: I3207e216466580eb7014aab93bf5a9c77eb77d13
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/50244
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-db.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dts b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
index 72ad60a68001..abca6311d93b 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
@@ -390,7 +390,7 @@
 
 &ap_sdhci0 {
 	status = "okay";
-	bus-width = <4>;
+	bus-width = <8>;
 	non-removable;
 };
 
-- 
2.31.1

